File: coding-practice/50 Coding Challenges part 1/JavaScript/scripts/problem9.js
// Coding challenge #9: Create a function that will convert from Fahrenheit to Celsius
function fahrenheit_to_celsius(fahrenheit){
    var celsius = (fahrenheit-32)/1.8;
    return celsius;
}

var fahrenheit = 27.9;
var celsius = fahrenheit_to_celsius(fahrenheit);
document.write("Celsius conversion of "+fahrenheit+" Fahrenheit is "+celsius+" Celsius");

File: coding-practice/50 Coding Challenges part 1/C#/problem9.cs
// coding challenge 9 : create a function that converts from Fahrenheit to Celsius
using System;
class problem9
{
    static float fahrenheit_to_celsius(float fahrenheit){
        float celsius = (fahrenheit - 32f)/1.8f;
        return celsius;
    }

    static void Main(string[] args){
        float fahrenheit = 40f;
        Console.WriteLine(fahrenheit + " Fahrenheit is equal to " + fahrenheit_to_celsius(fahrenheit) + " Celsius");
    }

}